tests/Listener/ShareCreatedEventListenerTest.php

```diff
@@ -31,9 +46,34 @@ protected function setUp(): void
         $this->mockLogger = $this->getMockBuilder(LoggerInterface::class)
             ->getMock();
 
+        $this->mockUser = $this->getMockBuilder(IUser::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->mockUserManager = $this->getMockBuilder(IUserManager::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->mockMailer = $this->getMockBuilder(IonosMailerService::class)
+            ->onlyMethods([
+                'send',
+            ])
+            ->getMock();
+
+        $this->mockL10N = $this->getMockBuilder(IL10N::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->mockUrlGenerator = $this->getMockBuilder(IURLGenerator::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
         $this->mockNode = $this->getMockBuilder(Node::class)
             ->getMock();
 
+        $this->mockTimestamp = $this->getMockBuilder(\DateTime::class)
```

Comment:
todo: mockTimestamp is undefined property. Has to be declared at some point before